BUILD.bazel

```
@@ -185,6 +185,7 @@ genrule(
     name = "all_generated_code",
     srcs = [
         "aten/src/ATen/native/native_functions.yaml",
+        "aten/src/ATen/native/tags.yaml"
```

I guess that was the source of the problem :)
```suggestion
        "aten/src/ATen/native/tags.yaml",
```